pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -106,6 +106,12 @@ type PodAutoscalerStatus struct {
 	// MetricsServiceName is the K8s Service name that provides revision metrics.
 	// The service is managed by the PA object.
 	MetricsServiceName string `json:"metricsServiceName"`
+
+	// DesiredScale shows the current desired number of replicas for the revision.
+	DesiredScale int32 `json:"desiredScale"`
+
+	// ActualScale shows the actual number of replicas for the revision.
+	ActualScale int32 `json:"actualScale"`
```

Comment:
For both of these it is probably meaningful to distinguish between unset and zero.  Can we make these `*int` and `omitempty`?

Reply:
Okay, done.